### ISSUE TITLE
App blocked from installing due to not having latest privacy protections

### ISSUE THREAD

**Issue by vishalscodes** [qw123wh/new-clock#40]
The app does not install while downloading the apk because it apparently doesn't follow the latest privacy protections.

Model: Honor 7x
Android Version: 9

![Screenshot_20240413_105334_com android vending](https://github.com/qw123wh/new-clock/assets/104351431/c054c272-95fd-46c5-8fff-d5dd2b8c20de)

Surprisingly, I didn't get this error while downloading com.chen.deskclock

**Comment by qw123wh** [comment]
It's a signature problem. However, for an official release, we have to wait a little longer

**Comment by BlackyHawky** [comment]
@vishalscodes Can you try it with the apk available [here](https://github.com/BlackyHawky/Clock/releases/tag/2.1) please?

**Comment by vishalscodes** [comment]
If I use the previous version, it doesn't install, but after uninstalling the other apk, it installs just fine. Also, with the debug apk, this issue persists.

**Comment by qw123wh** [comment]
Yes, uninstalling the previous version of the app is necessary. 
However, the important thing is not to receive errors installing the release version.
If in a few weeks we can also fix on Fdroid it will be perfect. 

Closing for to problem solved. I also invite you to follow the new reposity

**Comment by BlackyHawky** [comment]
> Also, with the debug apk, this issue persists.

I'm guessing that if you deactivate Google Play Protect, the debug version of the app will install correctly?

Does this problem also occur with debug versions of other applications such as [Heliboard](https://github.com/Helium314/HeliBoard/releases/tag/v1.3)?

In any case, the main thing is that this problem has been solved with the release version.

**Comment by vishalscodes** [comment]
@BlackyHawky I had to deactivate Google Play Protect to install the debug apk. I have not tried installing debug apps of apps like heliboard.

**Comment by BlackyHawky** [comment]
@vishalscodes Do you want to install the debug version of Heliboard just to see if the message appears?
Also, when you click on "more details", what does it say?

**Comment by vishalscodes** [comment]
@BlackyHawky Nope. When I click on "More details", it gives me an option to install without scanning. That is how I got the old version to install.


https://github.com/qw123wh/new-clock/assets/104351431/68662ab0-332d-4809-ae12-cf7cb1f9d5f7

**Comment by BlackyHawky** [comment]
Really strange message.
I don't know if it's worth spending time trying to solve this problem knowing that the release version installs correctly.
In any case, thanks for testing.

**Comment by qw123wh** [comment]
> Really strange message.
> I don't know if it's worth spending time trying to solve this problem knowing that the release version installs correctly.
> In any case, thanks for testing.

There is no problem.

There were practically no errors in the debug version never released. 

In the old release version it also gave me the same error, but it was simply a signature problem, which was not recognized by Google's Play Protect.
